src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -382,6 +382,44 @@ paths:
           description: request specified an invalid format
         '500':
           description: internal server error
+  /token-certificates/{hash}/import:
+    post: # ok
+      tags:
+        - security server
+      summary: import an existing certificate by cert hash
+      operationId: importExistingCertificate
+      description: Imports certificate from a token to the system
+      parameters:
+        - in: path
+          name: hash
+          description: SHA-1 hash of the certificate
+          required: true
+          schema:
+            type: string
+            format: text
+            minLength: 1
+            maxLength: 255
+      responses:
+        '201':
+          description: certificate imported
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CertificateDetails'
```

Comment:
Should return TokenCertificate instead. And same for other `token-certificates` endpoints.

Reply:
Fixed!